src/service/action-impl.js

```diff
@@ -415,6 +415,24 @@ export class ActionService {
 
   /**
    * Triggers the specified event on the target element.
+   * TODO(#32756): Rename this to `trigger` once all invocations have been
+   * updated to use the combined method.
+   * @param {!Element} target
+   * @param {string} eventType
+   * @param {?ActionEventDef} event
+   * @param {!ActionTrust} trust
+   * @param {?JsonObject=} opt_args
+   * @return {boolean} true if the target has an action.
+   */
+  triggerWithDispatch(target, eventType, event, trust, opt_args) {
```

Comment:
Up to you how you want to organize this across PRs, but the unit tests in `test-action.js` should eventually reflect expecting the custom event to fire.

Reply:
I noticed some of them actually test the error, while most assert that a stubbed "trigger" method is called, so I'll make adjustments accordingly